lib/common/presentation/widgets/genre_chip.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+import 'package:movie_app/common/presentation/build_context_extensions.dart';
+
+class GenreChip extends StatelessWidget {
+  final String name;
+  const GenreChip({
+    super.key,
+    required this.name,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      decoration: BoxDecoration(
+        color: context.appColors.genreTagBackground,
+        borderRadius: BorderRadius.circular(4),
+      ),
+      child: Padding(
+        padding: const EdgeInsets.symmetric(
+          horizontal: 8,
+          vertical: 4,
+        ),
+        child: Text(
+          name,
+          style: const TextStyle(color: Colors.white),
```

Comment:
Bilo bi dobro definirati nekoliko glavnih tekst stilova (inače se to iščita iz figme) u app_text_styles.dart i postaviti im vrijednosti i onda da uzimaš direktno taj textStyle gdje ti treba i eventualno mu namjestiš boju ako nije u onoj defaultnoj boji koju staviš za cijelu aplikaciju.